publish.gradle

```diff
@@ -47,43 +47,55 @@ configure(subprojects.findAll({ it.name in ["persistence", "synchronization", "d
                 }
             }
             publications {
-
                 pr.android.libraryVariants.all { variant ->
-                    def flavored = !variant.flavorName.isEmpty()
-                    def variantArtifactId = flavored ? variant.flavorName.replace('_', '-') : project.name
-                    variantArtifactId += variant.getBuildType().name == 'release' ? '' : variant.getBuildType().name.capitalize()
-
-                    def javaDocDestDir = file("${buildDir}/docs/javadoc ${flavored ? variantArtifactId : ""}")
 
-                    def sourceDirs = variant.sourceSets.collect {
-                        it.javaDirectories // includes Kotlin sources
+                    // Only publish full release variants
+                    def isDebug = variant.getBuildType().name == 'debug'
+                    def isFlavored = !variant.flavorName.isEmpty()
+                    def flavor = variant.flavorName.replace('_', '-')
+                    def isMock = isFlavored && flavor.contains('Mock')
+                    if (isDebug || isMock) {
+                        return
```

Comment:
Currently I only see a reason why we need to publish the FullRelease variants for both, "cyface" and "movebis".